doc/reference/resource_management/index.rst

```diff
@@ -84,5 +84,23 @@ state.
    :c:type:`onoff_manager`, with only a small reduction in functionality
    (primarily no support for the monitor API).
 
+.. _resource_mgmt_onoff_delayed:
+
+Delayed On-Off Manager
+**********************
+
+A delayed on-off manager is an extension to :ref:`resource_mgmt_onoff` which
+allows scheduled requests to the service. Manager takes the best effort to
+ensure that service will be in the on state at requested absolute or relative
+time. It takes into account service transision times. Start time should be a
+sum of maximum serivce startup time and maximum interrupt latency of the system
```

Comment:
I don't see what interrupt latency has to do with this.  The service should provide the maximum time required to initiate and complete a transition, assuming nothing delays it.  The application should be responsible for adjusting that based on its perception of any relevant delays that are  outside the control of the on-off service, whether they're due to interrupts or timeslicing or priority threads or whatever.

I think what you wrote here can be interpreted that way, but this PR seems to lack the extension to the on-off service so the maximum delays can be provided as input to that calculation.  With that, this could be made more clear, e.g. "the value of `delay_on` returned by `onoff_service_get_delays(mgr, &delay_on, &delay_off)` plus any notification delay resulting from other system activities".

Though I suspect most on-off service transitions are asynchronous, so the completion won't be affected by other things going on, only the notification of completion might be delayed and probably not by very much.